assets/src/application/ui/calendars/dateDisplay/BiggieCalendarDate/style.css

```diff
@@ -0,0 +1,185 @@
+/** -------------------------------------------------------------------------
+ * Event Espresso BiggieCalendarDate Stylesheet
+ * © 2020 Event Espresso
+ * ------------------------------------------------------------------------- */
+
+.ee-biggie-calendar-date-bg {
+	box-sizing: border-box;
+	color: var(--ee-color-white);
+	height: 100%;
+	padding: var(--ee-padding-tiny) var(--ee-padding-micro);
+	position: absolute;
```

Comment:
Which one is the relative parent for this?

Reply:
Guess some of these styles are too presumptuous about where this is going to be used... 
should maybe move this kind of control into the parent component, or allow it to be overridden via props